src/client/pythonEnvironments/base/locatorUtils.ts

```diff
@@ -87,6 +88,13 @@ export async function getEnvs(iterator: IPythonEnvsIterator): Promise<PythonEnvI
                 listener.dispose();
             } else {
                 const { index, update } = event;
+                if (envs[index] === undefined) {
+                    traceVerbose(
+                        `Updates sent for an env which was classified as invalid earlier, currently not expected, ${JSON.stringify(
+                            update,
+                        )}`,
+                    );
```

Comment:
This is one of the dumb parts of using a formatter.  This should all be one line.  In lieu of that, consider:

```suggestion
                    const json = JSON.stringify(update);
                    traceVerbose(`Updates sent for an env which was classified as invalid earlier, currently not expected, ${json}`);
```